### PR TITLE
Fixed typos across fission repo

### PR DESCRIPTION
**Pull request by gauravgahlot**
This PR fixes all the typos (99.99% 😄 ) in the fission repository. 

This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1832)

### CODE REVIEW

**Comment by gauravgahlot** [comment]
@vishal-biyani If the PR looks okay, we can avoid any _new_ PRs submitted for typo corrections. 😄

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ fission	fission  	1       	2020-05-19 16:31:46.947562 +0530 IST	success	fission-
 Also you should see the Fission services deployed and running:
 
 ```
-$ kubectl get pods -nfission
+$ kubectl get pods -n fission
```
Comment (vishal-biyani) [inline CONTRIBUTING.md:109]: `-nfission` is also legit, but what you did is also fine.

**Review by vishal-biyani — APPROVED** [review]

**Comment by vishal-biyani** [comment]
Thanks for one PR to rule dem all and welcome to Fission, your first PR is in @gauravgahlot ! 🎉 🎊

**Comment by chetanpdeshmukh** [comment]
Hello @gauravgahlot ,
Thanks again for the PR!

Thank you for contributing to Fission project. Could you please fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLSdhxtWL5N2PU7MYC2IRRAzYw-CrG9QtsYNvkiiOPVEQj8i51w/viewform?usp=sf_link), so we can send you the well deserved awesome swags :)

Team Fission

**Pull request merged**